hasura/metadata/tables.yaml

```diff
@@ -166,54 +166,6 @@
       - ethereum_address
       remote_schema: backend
     name: daohausMemberships
-  select_permissions:
-  - role: player
-    permission:
-      columns:
-      - availability_hours
-      - enneagram
-      - ethereum_address
-      - id
-      - playerTypeId
-      - rank
-      - role
-      - scIdentityId
-      - timezone
-      - totalXp
-      - tz
-      - username
-      filter: {}
-  - role: public
-    permission:
-      columns:
-      - availability_hours
-      - enneagram
-      - ethereum_address
-      - id
-      - rank
-      - role
-      - timezone
-      - totalXp
-      - tz
-      - username
-      filter: {}
-  update_permissions:
-  - role: player
-    permission:
-      columns:
-      - availability_hours
-      - enneagram
-      - playerTypeId
-      - role
-      - timezone
-      - tz
-      - username
-      filter:
-        id:
-          _eq: X-Hasura-User-Id
-      check: {}
-      set:
-        id: x-hasura-User-Id
```

Comment:
What are all these permission updates? Doesn't seem related to the timezone column

Reply:
Ugh.. there must be a bug in Hasura or something, as these permissions keep getting dropped when I make schema changes.